ui/apps/platform/src/Containers/Dashboard/Widgets/DeploymentsAtMostRiskTable.tsx

```diff
@@ -43,31 +44,38 @@ function DeploymentsAtMostRiskTable({
                 </Tr>
             </Thead>
             <Tbody>
-                {deployments.map(({ id, name, cluster, namespace, priority }) => (
-                    <Tr key={id}>
-                        <Td className="pf-u-pl-0" dataLabel={columnNames.deployment}>
-                            <Link to={riskPageLinkToDeployment(id, name, searchFilter)}>
-                                <Truncate position="middle" content={name} />
-                            </Link>
-                        </Td>
-                        <Td width={45} dataLabel={columnNames.resourceLocation}>
-                            <span>
-                                in &ldquo;
-                                <Link
-                                    to={`${networkBasePath}/${id}`}
-                                >{`${cluster} / ${namespace}`}</Link>
-                                &rdquo;
-                            </span>
-                        </Td>
-                        <Td
-                            width={20}
-                            className="pf-u-pr-0 pf-u-text-align-center-on-md"
-                            dataLabel={columnNames.riskPriority}
-                        >
-                            {priority}
-                        </Td>
-                    </Tr>
-                ))}
+                {deployments.map(({ id, name, cluster, namespace, priority }) => {
+                    const queryString = getQueryString({
```

Comment:
What I don't really like about this approach is the fact that if Network Graph's URL architecture changes, constructing links this way would cause issues. I think a more secure way of creating links would be to have some sort of helper function or hook within the Network Graph directory that we could import into other sections and construct links that way. That way we wouldn't have to know details about how to structure these URLs. We could supply the values as parameters to the helper functions or hooks and it would spit out the links we need

Reply:
Leaving a comment to come back to

Reply:
I think a helper function makes more sense. I'd go ahead and abstract it out now.

Reply:
Doing it in a follow-up draft PR makes more sense. I want to ensure everyone agrees on the approach (separately from this bug fix). Refactoring this bug fix should be trivial after